gloo/docs/user_guides/gateway/configuring_route_options/advanced_routing_action.md

```diff
@@ -195,11 +203,134 @@ apiVersion: gateway.solo.io/v1
               namespace: gloo-system
 {{< /highlight >}}
 
-### Multiple Destination {#multi}
+
+#### Kubernetes destination {#dest-kube}
+
+If you are running Gloo in a Kubernetes cluster, it is possible to directly specify 
+[Kubernetes Services](https://kubernetes.io/docs/concepts/services-networking/service/) as routing destinations. 
+The `kube` destination type has two required fields: `ref` is a [ResourceRef]({{% ref "/v1/github.com/solo-io/solo-kit/api/v1/ref.proto.sk#resourceref" %}}) 
+to the service that should receive traffic and `port` is the port on which the service is listening.
```

Comment:
I assume the port is a number field, and that the port must be one of the existing defined in the service's associated k8s Service manifest. Helpful to have a little more detail on any constraints.

Reply:
Updated